src/source/libscalene.cpp

```diff
@@ -158,22 +159,19 @@ class MakeLocalAllocator {
     const auto allocSize = len + sizeof(ScaleneHeader);
     buf = get_original_allocator()->malloc(ctx, allocSize);
     auto *header = new (buf) ScaleneHeader(len);
+    class Nada {};
 #else
     auto *header = (ScaleneHeader *)get_original_allocator()->malloc(ctx, len);
 #endif
     assert(header);                  // We expect this to always succeed.
-    if (len <= PYMALLOC_MAX_SIZE) {  // don't count allocations pymalloc passes
+    if (! m.wasInMalloc() || len == SampleHeap<1, HL::NullHeap<Nada>>::NEWLINE ) {  // don't count allocations pymalloc passes
```

Comment:
An attempt at a check for if we were missing `NEWLINE`s, should be removed

Reply:
@sternj still here?